src/cc-index.ts

```diff
@@ -235,6 +238,9 @@ export class CCIndex {
                              ccIndexPageURL? : string) {
 
         lookupURL = lookupURL.replace("https://", "").replace("http://", ""); // remove http(s)
+        if (! CCIndex.defaultCCIndex) {
+            CCIndex.initDefaultIndex();
+        }
```

Comment:
I don't like this. Creating an instance of `CCIndex` with the required parameters would be a lot cleaner.